source/dh_override.c

```diff
@@ -56,7 +70,7 @@ DH *d2i_DHparams(DH **a, const unsigned char **pp, long length) {
 }
 
 int DH_check(DH *dh, int *codes) {
-    assert(openssl_DH_is_valid(dh));
+    assert(dh != NULL);
```

Comment:
This part is unclear to me. `DH_size` calls `openssl_DH_is_valid` but here we just check null-ness. Why is that? May be leave a one line comment as to why this change was necessary?

Reply:
I created this issue about removing some of the validity functions https://github.com/awslabs/aws-verification-model-for-libcrypto/issues/17